GOVERNANCE.adoc

```diff
@@ -0,0 +1,71 @@
+This document describes the governance rules of the ServiceTalk project/organization. It is meant to be followed by all
+the repositories / sub-projects in the organization.
```

Comment:
nit: in the sentence before you did not use ` / ` but `/` (not enclosed by whitespaces)

Reply:
corrected